scripts/lib/language-helper.ts

```diff
@@ -0,0 +1,9 @@
+import Data from "sf-core/global/data";
+
```

Comment:
It is practically better to do something like `Data.getStringVariable('language') || Data.setStringVariable('language', 'en');` // or whatever your default language is to not worry about non existing cases